build/pluginctl/logs.go

```diff
@@ -15,7 +15,7 @@ import (
 )
 
 const (
-	logsPerPage     = 100 // logsPerPage is the number of log entries to fetch per API call
+	logsPerPage     = 100 // logsPerPage is the number of log entries to fetch per PluginAPI call
```

Comment:
Note to reviewer - these `API` to `PluginAPI` text changes were accidental and are already fixed in `main`.